specs/Modal.helpers.spec.js

```diff
@@ -15,12 +38,16 @@ export default () => {
     describe("with tabbable descendents", () => {
       let elem;
       beforeEach(() => {
+        registerTestComponent();
         elem = document.createElement("div");
         document.body.appendChild(elem);
       });
 
       afterEach(() => {
-        document.body.removeChild(elem);
+        document
+          .querySelectorAll("test-web-component")
+          .forEach(node => node.remove());
+        elem.remove();
```

Comment:
Maybe it would be better to split these tests.

Reply:
I separated them into a separate `describe`. Does that work for you?